### PR TITLE
Adding jali to docker build

### PR DESCRIPTION
**Pull request by nray**

### CODE REVIEW

**Comment by cmsquared** [comment]
Looks like we need to bump the CMake version for Jali to build

**Comment by nray** [comment]
@cmsquared and @ktsai7 : Can one of you go through this PR and approve it ?

File: ubuntu [hunk 1]
```diff
@@ -20,3 +30,46 @@ USER portage
...
+     -D ENABLE_ExodusII=yes \
+     -D ENABLE_Tests=yes ..
+RUN make VERBOSE=1 -j2 && make test
+RUN make install
+RUN ls $MSTK_DIR
+
+WORKDIR ../..
+RUN git clone https://github.com/lanl/jali.git
+WORKDIR jali
+#RUN git checkout 1.0.5
```
Comment (cmsquared) [inline ubuntu]: Not specifying a version here could be fragile going forward...

**Review by cmsquared — APPROVED** [review]

File: ubuntu (hunk 1 above)
Comment (nray) [reply ubuntu]: Yes, you are right. Checking out v1.1.3 now. If this builds, we are good to go.

**Pull request merged**